optuna/visualization/_pareto_front.py

```diff
@@ -0,0 +1,216 @@
+import json
+from typing import List
+from typing import Optional
+
+import optuna
+from optuna._experimental import experimental
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+from optuna.trial import TrialState
+from optuna.visualization._plotly_imports import _imports
+
+
+if _imports.is_successful():
+    from optuna.visualization._plotly_imports import go
+
+_logger = optuna.logging.get_logger(__name__)
+
+
+@experimental("2.4.0")
+def plot_pareto_front(
+    *,
+    study: Study,
```

Comment:
`study` could perhaps be relaxed to positional. That'd also be more consistent with the existing functions. Not a strong opinion since it's experimental.